package-lock.json

```diff
@@ -1,7 +1,8 @@
 {
   "name": "parse-stitch-query",
-  "version": "1.0.2",
+  "version": "1.0.3",
```

Comment:
Oops - looks like this didn't get updated with the last commit, which release 1.0.3.